application/run-cron.sh

```diff
@@ -15,6 +15,9 @@ chown -R www-data:www-data \
 
 # Run database migrations
 /data/yii migrate --interactive=0
+if [ $? -ne 0 ]; then
+    exit $?
```

Comment:
Can you read the `$?` more than once for a given execution? I simply don't know whether this will work, or whether the second `$?` will be returning something about the `if` comparison.

You may know better/already. Feel free to proceed as you see fit.

Reply:
I don't know, actually. That's a great observation.